subsys/testsuite/coverage/coverage_rodata.ld

```diff
@@ -3,7 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#ifdef CONFIG_64BIT
+. = ALIGN(8);
```

Comment:
Good catch 👍

Reply:
@fabiobaltieri, @ycsin - given that both of you have run into alignment-related issues (possibly on multiple platforms?) I wonder if this is potentially fixed already or if this was just a symptom of a larger issue. Would be great to verify.

Reply:
The one I bumped into is still failing (`west build -t run -p -b qemu_riscv32_xip -T tests/kernel/xip/arch.common.xip`), I think it's a larger issue.